pkg/backend/filestate/backend.go

```diff
@@ -622,9 +598,9 @@ func GetLogsForTarget(target *deploy.Target, query operations.LogQuery) ([]opera
 }
 
 func (b *localBackend) ExportDeployment(ctx context.Context,
-	stackRef backend.StackReference) (*apitype.UntypedDeployment, error) {
+	stk backend.Stack) (*apitype.UntypedDeployment, error) {
```

Comment:
Nit: `stk` => `stack`?

Reply:
Can't: doing so shadows the `stack` package, and that package is required by this function.